src/Tempest/Support/src/HtmlHelper.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tempest\Support;
+
+final class HtmlHelper
+{
+    public static function createTag(string $tag, array $attributes = [], ?string $content = null): string
```

Comment:
Not really a fan of static helpers, I'd prefer to move to instantiable, immutable classes. That's why I recently refactoered `PathHelper` as well

Reply:
While I'm not a fan of them either, sometimes they make sense I think.

In this case, none of the methods in that helper class would act on an existing string. `createTag` could be a static constructor, but then the class would basically be an empty `Stringable` instance...

Do you have suggestions?

Reply:
Actually... https://github.com/tempestphp/tempest-framework/pull/842